### DISCUSSION TITLE
Welcome to AP_Project Discussions!

### DISCUSSION THREAD

**Discussion by MahdiHaeri** [MahdiHaeri/AP_Project#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amirhossein2831** [comment]
hey man, I'm amirhossein motaghian who love to communicate  with other developer help them and also learn from other guy.

**Reply by seyedalida** [reply]
Ahsant!

**Comment by seyedalida** [comment]
Ahsant!
